screens/CheckoutScreen.js

```diff
@@ -0,0 +1,107 @@
+import React from "react";
+import { BASE } from "../lib/common";
+
+import {
+  AsyncStorage,
+  StyleSheet,
+  TextInput,
+  View,
+  Button,
+  Text
+} from "react-native";
+
+export default class CheckoutScreen extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      customerName: "",
+      customerId: 0,
+      currPoints: 0,
+      cart: [],
+      totalPrice: 0,
+      totalPoints: 0
+    };
+  }
+
+  async componentDidMount() {
+    const customerId = await AsyncStorage.getItem("customerId");
+    this.setState({ customerId: customerId });
+
+    this.getUser("Customers", customerId).then(customerRecord => {
+      if (customerRecord) {
+        let currPoints = customerRecord["fields"]["points"];
+        let name =
```

Comment:
In the future, I think we can make a 'Name' via formula in Airtable and replace this (:

Reply:
Removing this file D: